javaharness/java/arcs/api/ShellApi.java

```diff
@@ -1,7 +1,27 @@
 package arcs.api;
 
-/** An API for Java to call Javascript (shell). */
-public interface ShellApi {
-  // Sends a message to Arcs JS shell.
-  String receive(String json);
-}
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class ShellApi {
```

Comment:
Good idea. Renamed to ArcsMessageSender.

It's really up to a design question: is a shell a must for Arcs? I personally prefer the answer is not as otherwise we'll need to explain what is a shell. So I renamed it.